include/deal.II/distributed/fullydistributed_tria.h

```diff
@@ -0,0 +1,237 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2008 - 2019 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE.md at
+// the top level directory of deal.II.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii_fullydistributed_tria_h
+#define dealii_fullydistributed_tria_h
+
+
+#include <deal.II/base/config.h>
+
+#include <deal.II/base/smartpointer.h>
+#include <deal.II/base/subscriptor.h>
+#include <deal.II/base/template_constraints.h>
+#include <deal.II/base/timer.h>
+
+#include <deal.II/distributed/p4est_wrappers.h>
+#include <deal.II/distributed/tria_base.h>
+
+#include <deal.II/grid/tria.h>
+
+#include <boost/range/iterator_range.hpp>
+
+#include <functional>
+#include <list>
+#include <set>
+#include <tuple>
+#include <type_traits>
+#include <utility>
+#include <vector>
+
+#ifdef DEAL_II_WITH_MPI
+#  include <mpi.h>
+#endif
+
+#ifdef DEAL_II_WITH_P4EST
```

Comment:
Do you need these headers? I think you can skip everything p4est related in this file.

Reply:
These were a copy of the includes in `tria.h`. I have removed them.